snaps_k8s/provision/ansible_configuration.py

```diff
@@ -232,7 +232,17 @@ def __kubespray(k8s_conf, base_pb_vars):
             consts.K8_CPU_PINNING_CONFIG,
             variables={'KUBESPRAY_PATH': config_utils.get_kubespray_dir(
                 k8s_conf)})
-
+                
+    ha_enabled = False
+    ha_configuration = config_utils.get_ha_config(k8s_conf)
+    if ha_configuration:
```

Comment:
This is ok but I'd like to see a function in config_utils that will simply return the boolean for ha_enabled.

Reply:
comments incorporated. Removed ha_enabled flag as it is no longer used after comments incorporation.